src/libraries/System.Composition.TypedParts/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <!-- .NETStandard,Version=v1.0 and .NETPortable,Version=v0.0,Profile=Profile259 are intentionally dropped from the package. -->
+  <Suppression>
```

Comment:
it seems like suppressions file is same for all the projects. can we have just one copy ?

Reply:
Considered that but as this is just temporary until we ship the NET6 packages I would prefer to keep the suppression files for each project and then clean them all together up in NET7.